webapp/components/List/List.vue

```diff
@@ -5,6 +5,7 @@
       :id="item.id"
       :key="item.id"
       :item="item"
+      :is-authenticated="isAuthenticated"
```

Comment:
⭐ For a delete button that is only visible to the author of the post.
❌ I was not able to find your edit button, did you implement it? Or did I oversee it?